### DISCUSSION TITLE
how to use the catogories and tags function like demo?

### DISCUSSION THREAD

**Discussion by visajiandrevv** [adityatelange/hugo-PaperMod#939] [Q&A] [answered]
i didn't find a introduction of the catogories and tags function,but i see they existed in the demo.can anyone tell me how to open it ?like archive function has a achive.md, are there something i should write in the tags.md or catogories.md?

**Comment by visajiandrevv** [comment, marked as answer]
i solved it .to use both u should create files in <code>/content/tags/index.md  and /content/categories/index.md</code>

And both write like
<pre>
---
title: "database"
layout: "categories"
---
</pre>

by the way i still confused that i did not find a into about this.

**Comment by r0hitsharma** [comment]
@visajiandrevv https://gohugo.io/content-management/taxonomies should explain this.
